doc/source/index.md

```diff
@@ -2,200 +2,226 @@
-{link-badge}`train/user_guide.html,"Train",cls=badge-light`
-{link-badge}`tune/user-guide.html,"Tune",cls=badge-light`
-{link-badge}`serve/tutorial.html,"Serve",cls=badge-light`
-{link-badge}`cluster/user-guide.html,"Clusters",cls=badge-light`
----
-**API reference**
+|<img src="./images/multi_model_serve.png" width="70%" loading="lazy">|
+|:--:|
+|Deployment patterns with Ray Serve.|
 
-<img src="images/api.svg" alt="api" height="40px">
-^^^^^^^^^^^^^
+See [model serving examples](https://docs.ray.io/en/latest/ray-overview/use-cases.html#model-serving) for use cases.
 
-Our API reference guide provides you with a detailed description of the different Ray APIs.
-It assumes familiarity with the key concepts and gives you information about functions, classes, and methods.
+### ML platform
 
-+++
+[Merlin](https://shopify.engineering/merlin-shopify-machine-learning-platform) is Shopify's ML platform built on Ray. It enables fast-iteration and [scaling of distributed applications](https://www.youtube.com/watch?v=kbvzvdKH7bc) such as product categorization and recommendations.
 
-{link-badge}`ray-references/api.html,"API References",cls=badge-light`
+|<img src="./images/shopify-workload.png" width="70%" loading="lazy">|
+|:--:|
+|Shopify's Merlin architecture built on Ray.|
 
----
-**Developer guides**
+Spotify [uses Ray for advanced applications](https://www.anyscale.com/ray-summit-2022/agenda/sessions/180) that include personalizing content recommendations for home podcasts, and personalizing Spotify Radio track sequencing.
 
-<img src="images/contribute.svg" alt="contribute" height="40px">
-^^^^^^^^^^^^^^^
+|<img src="./images/spotify.png" width="70%" loading="lazy">|
+|:--:|
+|How Ray ecosystem empowers ML scientists and engineers at Spotify.|
 
-You need more information on how to debug or profile Ray?
-You want more information about Ray's internals?
-Maybe you saw a typo in the documentation, want to fix a bug or contribute a new feature? 
-Our developer guides will help you get started.
+See [ML platform examples](https://docs.ray.io/en/latest/ray-overview/use-cases.html#ml-platform) for use cases.
 
-+++
 
-{link-badge}`https://docs.ray.io/en/master/ray-contribute/getting-involved.html,"Developer Guides",cls=badge-light`
+## Getting involved
+
+Ray is a framework for distributed applications and also an active community of developers, researchers, and folks that love machine learning.
+Here's how to get involved with the Ray community:
+
+```{include} _includes/_contribute.md
+```
+
+If you're interested in contributing to Ray, see the
+[contributing guide for the current release](ray-contribute/getting-involved)
+or the
+[latest version of our contributing guide](https://docs.ray.io/en/latest/ray-contribute/getting-involved.html)
+to read about the contribution process and see what you can work on.
+
+```{image} https://github.com/ray-project/ray/raw/master/doc/source/images/ray_header_logo.png
```

Comment:
you moved the header and all logos to the bottom (see rendered page). I don't think that's what we want. The badges we can discuss, but a prominent logo should still be featured up top.

Reply:
Let me share why I removed it from the top. I thought that it was generally a bad user experience to have to scroll through graphics to get to content. The logo is also in the upper left hand corner, so the larger logo at the top is redundant.

I referenced other docs home pages to check my bias:

- https://docs.databricks.com/
- https://modal.com/docs/guide (their docs have been coming up in conversations recently)
- https://stripe.com/docs
- https://www.twilio.com/docs
- https://platform.openai.com/docs/introduction
- https://docs.grid.ai/

Those examples all have the logo in the upper left corner. WDYT, @maxpumperla? Let me know if you think my examples are too skewed.

Reply:
Hey! I like it! I do agree that we do not need the logo on top of the page since we already have the Ray logo on the horizontal navigation bar. We also have the text "Ray 3.0.0" on the left vertical navigation, so putting the Ray logo again in the page might be redundant.

Reply:
Yes, I had the same feeling of splitting out the ML Workloads into a separate page. Thanks, @simran-2797! I'll file an issue to "freeze" the top nav.

Reply:
@angelinalg ok, understood. removing the logo is fine. I just thought it looked silly to put it at the bottom - better to remove altogether.

Reply:
I removed the logo. Thanks, @maxpumperla.